uportal-war/src/main/java/org/jasig/portal/groups/pags/dao/EntityPersonAttributesGroupStore.java

```diff
@@ -317,7 +317,7 @@ private GroupDefinition initGroupDef(IPersonAttributesGroupDefinition group) {
             TestGroup tg = new TestGroup();
             Set<IPersonAttributesGroupTestDefinition> tests = testGroup.getTests();
             for(IPersonAttributesGroupTestDefinition test : tests) {
-                IPersonTester testerInst = initializeTester(test.getTesterClassName(), test.getAttributeName(), test.getTestValue());
+                IPersonTester testerInst = initializeTester(test);
                 tg.addTest(testerInst);
```

Comment:
I think I agree with @apetro here. Check for null, log properly if is null, continue. Worst case is they are not part of a group they should be so they don't see something they should have access to.